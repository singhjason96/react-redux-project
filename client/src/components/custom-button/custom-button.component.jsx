import React from 'react';
import { CustomButtonContainer } from './custom-button.styles.jsx';


export const CustomButton = ({ children, ...props }) => (
    <CustomButtonContainer {...props}>{children}</CustomButtonContainer>
);

export default CustomButton;