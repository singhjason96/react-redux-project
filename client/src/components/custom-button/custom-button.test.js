import React from 'react';
import { shallow } from 'enzyme';
import { CustomButton } from './custom-button.component';
import "../../setupTests";


it('renders custom button component', () => {
    expect(shallow(<CustomButton />)).toMatchSnapshot();
})

