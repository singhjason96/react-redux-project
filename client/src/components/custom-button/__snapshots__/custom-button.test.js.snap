// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders custom button component 1`] = `ShallowWrapper {}`;
